server/resolvers/FeedTime.js

```diff
@@ -25,7 +25,16 @@ const Feeder = {
         })
     },
     async updateFeedTime(parent, { _id, ...args }, { models }) {
-      // todo
+      return models.FeedTime.findByIdAndUpdate(_id, args, {
+        new: true,
+        upsert: true
```

Comment:
Looks good overall, but this line particularly is not necessary. For context the options here do as follows:

`new`: whether to return the modified FeedTime document or the original one
`upsert`: whether to insert the data passed in if the document doesn't already exist

Here, we'd probably like to return an error if the `_id` doesn't refer to any document as opposed to creating a new document.

Reply:
Line 30 is what I'm referring to by the way